Nfield.Quota/QuotaLevelDefinition.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Nfield.Quota
 {
-    public class QuotaLevelDefinition : IEquatable<QuotaLevelDefinition>
+    public sealed class QuotaLevelDefinition : IEquatable<QuotaLevelDefinition>
```

Comment:
this is a breaking change, we would have to do a major version bump.

Reply:
Version changed to 4.0